WcaOnRails/config/initializers/ruby_money.rb

```diff
@@ -1,3 +1,7 @@
 # frozen_string_literal: true
 
 Money.locale_backend = :i18n
+
+eu_bank = EuCentralBank.new
+eu_bank.update_rates
```

Comment:
Does putting this here mean that currency exchange rates only get updated whenever the server is getting deployed?

Reply:
indeed. The `eu_bank` github readme also recommends updating it every time we convert, but I don't know if that would be too slow.